core/storage/topic/gae_models.py

```diff
@@ -316,8 +316,8 @@ def get_export_policy(cls) -> Dict[str, base_models.EXPORT_POLICY]:
             'next_subtopic_id': base_models.EXPORT_POLICY.NOT_APPLICABLE,
             'language_code': base_models.EXPORT_POLICY.NOT_APPLICABLE,
             'meta_tag_content': base_models.EXPORT_POLICY.NOT_APPLICABLE,
-            'page_title_fragment_for_web': (
-                base_models.EXPORT_POLICY.NOT_APPLICABLE),
+            'page_title_fragment_for_web':
```

Comment:
FYI, OK to keep the parens in this case (we typically do that at the end of a line to indicate that the line continues).

Reply:
Done.